src/test/java/org/prebid/server/validation/BidderParamValidatorTest.java

```diff
@@ -232,6 +235,42 @@ public void validateShouldReturnValidationMessagesWhenFacebookExtNotValid() {
         assertThat(messages.size()).isEqualTo(1);
     }
 
+    @Test
+    public void validateShouldNotReturnValidationMessagesWhenOpenxImpExtIsOk() {
+        // given
+        final ExtImpOpenx ext = ExtImpOpenx.builder()
+                .customParams(Collections.singletonMap("foo", "bar"))
+                .customFloor(0.2f)
+                .delDomain("se-demo-d.openx.net")
+                .unit("2222")
+                .build();
+        final JsonNode node = mapper.convertValue(ext, JsonNode.class);
+
+        // when
+        final Set<String> messages = bidderParamValidator.validate(OPENX, node);
+
+        // then
+        assertThat(messages).isEmpty();
+    }
+
+    @Test
+    public void validateShouldReturnValidationMessagesWhenOpenxExtNotValid() {
+        // given
+        final ExtImpOpenx ext = ExtImpOpenx.builder()
+                .customParams(Collections.singletonMap("foo", "bar"))
+                .customFloor(0.2f)
+                .delDomain("se-demo-d.openx.net")
+                .unit("not-numeric")
+                .build();
+        final JsonNode node = mapper.convertValue(ext, JsonNode.class);
+
+        // when
+        final Set<String> messages = bidderParamValidator.validate(OPENX, node);
+
+        // then
+        assertThat(messages.size()).isEqualTo(1);
+    }
+
```

Comment:
It seems we didn't do it before, but in openx.json we have rules for 4 fields. Maybe it will be good to write tests to check that fields are correct types and match patterns. For example delDomain should be string and match "\\.[a-zA-Z]{2,3}$" pattern, same can be done for unit field. Check required fields   "required": ["unit", "delDomain"]. Or we do not do it?

Reply:
We did not test bidders schema's that carefully. Not sure if we should. Here we ensuring that schema exist and briefly checking if it works. Let's discuss it.

Reply:
After discussion we agreed that these tests will cover third party library logic, not our. There no needs in these test